architecture/adr-list/adr-008-ledger-resources.md

```diff
@@ -230,140 +190,217 @@ Example using the Veramo CLI:
+            "resourceType": "CL-Schema",
+            "alsoKnownAs": [
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": ""
+                },
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": "Alternative URI description"
+                }
+            ]
+        }
+    ]
+}
 ```
 
-#### QueryGetResourceRequest
+#### QueryResourceRequest
 
-* Collection ID: String - an identifier of linked DIDDoc
-* ID: String - unique resource id
+* `Collection ID`: String - an identifier of linked DIDDoc
+* `ID`: String - unique resource id
 
 Example:
 
 ```jsonc
 { 
-  "collectionId": "DAzMQo4MDMxCjgwM",
-  "id": "bb2118f3-5e55-4510-b420-33ef9e1726d2"
+  "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+  "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f"
 }
 ```
 
-#### QueryGetResourceResponse
+#### QueryResourceResponse
 
-* Resource: [Resource](#resource)
+* Returns `Resource` with a given `collection ID` and `ID`.
 
 Example:
 
 ```jsonc
-{ "resource":  <Resource> }
+{
+    "resource": {
+        "resource": {
+            "data": "eyJ0ZXN0IjogInRlc3QifQ=="
+        },
+        "metadata": {
+            "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+            "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+            "name": "PassportScheme",
+            "version": "1.0",
+            "resourceType": "CL-Schema",
+            "alsoKnownAs": [
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": ""
+                },
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": "Alternative URI description"
+                }
+            ],
+            "media_type": "application/json",
+            "created": "2023-01-21T20:02:55.664985039Z",
+            "checksum": "a7cd6c222ea5fc1463c0ca3f70b93035196c8c4f34d89181ff5086bd7b58bfff"
+        }
+    }
+}
 ```
 
-#### QueryGetAllResourceVersionsRequest
+#### QueryResourceMetadataRequest
 
-* Collection ID: String - an identifier of linked DIDDoc
-* Name: String
-* ResourceType: String
+* `Collection ID`: String - an identifier of linked DIDDoc
+* `ID`: String - unique resource id
 
 Example:
 
 ```jsonc
 { 
-  "collectionId":  "DAzMQo4MDMxCjgwM",
-  "name":          "PassportSchema",
-  "resourceType":  "CL-Schema"
+  "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+  "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f"
 }
 ```
 
-#### QueryGetAllResourceVersionsResponse
-
-* Resources: [ResourceHeader\[\]](#resourceheader)
+#### QueryResourceMetadataResponse
 
-Example:
+Returns `resource's metadata` with a given `collection ID` and `ID`.
 
 ```jsonc
-{ "resources":  [<ResourceHeader1>, <ResourceHeader2>] }
+{
+    "resource": {
+        "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
```

Comment:
the same question here. I think we are responding in snake case, could you please check it also here?

Reply:
Done! ✅